scipy/optimize/tests/test_linprog.py

```diff
@@ -2316,6 +2316,67 @@ def test_mip6(self):
 
         np.testing.assert_allclose(res.fun, 1854)
 
+    @pytest.mark.slow
+    @pytest.mark.timeout(600)
```

Comment:
```suggestion
    @pytest.mark.xslow
```
Actually, tests that aren't critical to run all the time that take longer than >>1s are usually marked xslow. Let's do that here.